File: 268. Missing Number.java
public class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        int output = 0;
        
        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] != (nums[i + 1] - 1)) {
                output = (nums[i] + 1);
                return output;
            }
        }
        
        if (nums[0] == 0) {
            return (nums[nums.length - 1]) + 1;
        }
        return nums.length - nums[nums.length - 1];
    }
}

File: 268. Missing Number.py
class Solution:
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = set(nums)
        
        for i in range(len(nums)):
            if i not in n:
                return i
        
        return len(nums)
